src/ws/ws_manager.rs

```diff
@@ -88,6 +98,25 @@ impl WsManager {
         };
         spawn(reader_fut);
 
+        {
+            let writer = writer.clone();
```

Comment:
nit: I think `let writer = Arc::clone(&writer)` is more idiomatic

Reply:
Done